extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -56,7 +56,7 @@
     private static final String RESOURCES_DIR = "resources";
     private static final String LIBS_DIR = "libs";
 
-    private static final String EXTENSION_STORE_URI = "extension.store.uri";
+    public static final String EXTENSION_STORE_URI = "extension.store.uri";
```

Comment:
Since this is a startup property, it could be cleaner to define it in StartupProperties. For example look at CONFIGSTORE_PROPERTY